assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/globalnetworkpolicies.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.calico.metadataCIDRs }}
+apiVersion: crd.projectcalico.org/v1
+kind: GlobalNetworkPolicy
+metadata:
+  name: block-metadata-access
+spec:
+  selector: !has(host-endpoint)
```

Comment:
Why `!has(host-endpoint)`? This isn't clear to me.

Reply:
I'll add a comment. This is because we want to block all `WorkloadEndpoint`s (so Pods), but not HostEndpoints (so host networking).

Reply:
Got it. Shouldn't we use `NetworkPolicy` then? IIUC this distinction is the main reason for having both GNPs and NPs.

Reply:
We must use GNP, because NPs are namespace specific and we want to block this traffic also for the new namespaces being created by the user and for all components.

Reply:
I added extra comments to document how it works.